Source/WPEFramework/Controller.cpp

```diff
@@ -183,6 +183,132 @@ namespace Plugin {
         return (result);
     }
 
+    uint32_t Controller::Persist()
+    {
+        ASSERT(_pluginServer != nullptr);
+
+        uint32_t result = _pluginServer->Persist();
+
+        // Normalise return code
+        if (result != Core::ERROR_NONE) {
+            result = Core::ERROR_GENERAL;
+        }
+
+        return result;
+
+    }
+
+    uint32_t Controller::Delete(const string& path)
```

Comment:
I see that the delete code is now different then it was previously in endpoint_delete. For all the moved code let's just keep the implementation that was already in there (didnt check the others that were move3d, so please check these as well)

Reply:
will check and update

Reply:
The update required for only Delete method, I removed Normalise call usage instead used older implementation. In Clone() method, the changes I added is just renaming variable to camel case